src/main/java/appeng/recipes/RecipeHandler.java

```diff
@@ -74,696 +31,15 @@
  */
 public class RecipeHandler implements IRecipeHandler
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "IRecipeHandler"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)